pkg/services/serviceaccounts/api/api.go

```diff
@@ -136,7 +135,7 @@ func (api *ServiceAccountsAPI) CreateServiceAccount(c *contextmodel.ReqContext)
 
 	serviceAccount, err := api.service.CreateServiceAccount(c.Req.Context(), c.OrgID, &cmd)
 	switch {
-	case errors.Is(err, database.ErrServiceAccountAlreadyExists):
```

Comment:
unfortunely we have two of the same errors here. `database` and `serviceaccounts` alreadyExists. Should refactor this for sure.
Should we check both perhaps?